packages/turf-helpers/index.js

```diff
@@ -535,19 +534,7 @@ export function isObject(input) {
     return (!!input) && (input.constructor === Object);
 }
 
-const RADIUS = 6378137;
-const FLATTENING_DENOM = 298.257223563;
-const FLATTENING = 1 / FLATTENING_DENOM;
-const POLAR_RADIUS = RADIUS * (1 - FLATTENING);
-
 /**
- * WGS84 Provides radius, FLATTENING, and POLAR_RADIUS constants from the reference ellipsoid.
- *
- * http://earth-info.nga.mil/GandG/wgs84/gravitymod/egm96/egm96.html
+ * Earth Radius used with the Harvesine formula and approximates using a spherical (non-ellipsoid) Earth.
  */
-export const wgs84 = {
-    RADIUS,
-    FLATTENING_DENOM,
-    FLATTENING,
-    POLAR_RADIUS,
-};
+export var earthRadius = 6371008.8;
```

Comment:
@stebogit Added `earthRadius` to `@turf/helpers`. (was suppose to be different PR).

Removed `wgs84` info since we aren't doing those types of calculations.

Ref: https://github.com/Turfjs/turf/issues/978